tests/libs/__mocks__/prisma.ts

```diff
@@ -1,18 +1,90 @@
+import { PrismockClient } from "prismock";
 import { beforeEach, vi } from "vitest";
-import { mockDeep, mockReset } from "vitest-mock-extended";
 
-import type { PrismaClient } from "@calcom/prisma";
+import logger from "@calcom/lib/logger";
+import * as selects from "@calcom/prisma/selects";
 
 vi.mock("@calcom/prisma", () => ({
   default: prisma,
   prisma,
-  availabilityUserSelect: vi.fn(),
-  userSelect: vi.fn(),
+  ...selects,
 }));
 
+const handlePrismockBugs = () => {
```

Comment:
I intend to create bugs in the repo for these. They should not be too hard to fix in there.